instance/tests/integration/integration_instance.py

```diff
@@ -293,19 +294,19 @@ def test_spawn_appserver(self):
         self.assert_instance_up(instance)
         self.assert_appserver_firewalled(instance)
         self.assertTrue(instance.successfully_provisioned)
-        self.assertTrue(instance.require_user_creation_success())
         for appserver in instance.appserver_set.all():
             self.assert_secret_keys(instance, appserver)
             self.assert_lms_users_provisioned(user, appserver)
             self.assert_theme_provisioned(instance, appserver, application)
 
     @shard(1)
+    @override_settings(INSTANCE_STORAGE_TYPE='s3')
     def test_betatest_accepted(self):
         """
         Provision an instance, spawn an AppServer and accepts the application.
         """
         OpenEdXInstanceFactory(
-            name='Integration - test_spawn_appserver',
+            name='Integration - test_betatest_accepted',
```

Comment:
@viadanna Thanks for making instance names match test names -- that's a small but awesome change! :D

Reply:
Yeah, this was necessary to prevent clashes in S3 bucket names during tests.